src/test/java/com/github/paweladamski/httpclientmock/HttpClientResponseBuilderTest.java

```diff
@@ -198,10 +199,22 @@ public void should_return_json_with_right_header() throws IOException {
                 .doReturnJSON("{foo:1}", Charset.forName("UTF-8"));
         HttpResponse login = httpClientMock.execute(httpGet("http://localhost:8080/login"));
 
+        System.out.println(ContentType.get(login.getEntity()));
```

Comment:
I think that instead of printing content type to console we should add assert verifying if the value is correct.

Reply:
i can remove the sysout, to bring back the code as before,because i have not changed this test context wise.